include/pmacc/PMaccConfig.cmake

```diff
@@ -141,6 +141,11 @@ if((NOT ALPAKA_ACC_GPU_CUDA_ENABLE) AND ALPAKA_ACC_GPU_CUDA_ONLY_MODE)
     message(WARNING "ALPAKA_ACC_GPU_CUDA_ONLY_MODE is set to OFF because cuda backend is not activated")
 endif()
 
+# includes and definitions are already added in cupla's & alpaka's
```

Comment:
I would hope that the CMake native CUDA language support would allow this. However, it does not yet support clang as native CUDA compiler at all. Furthermore, I do not know how it handles `.cpp` files containing CUDA code. Currently `alpaka_add_executable` enforces CUDA compilation of all extensions.